code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1458,3 +1458,58 @@
 	drink_desc = "A sawed-off cola bottle filled with Fernet Cola. You can hear cuarteto music coming from the inside."
 	taste_description = "low class heaven"
 	remove_nutrition = 1
+
+/datum/reagent/consumable/ethanol/gimlet
+    name = "Gimlet"
+    id = "gimlet"
+    description = "A sharp cocktail dating back to the 19th century. Gin and lime, nothing else."
+    color = "#DEF8AB" // rgb (222, 248, 171)
+    alcohol_perc = 0.3
+    drink_icon = "gimlet"
+    drink_name = "Gimlet"
+    drink_desc = "There are debates on whether this drink should be half gin and half lime, or three parts gin and one part lime. All you know is, it's alcohol."
+    taste_description = "sharpness"
```

Comment:
You can't see it on the github diffs, but some of the new drinks are indented in the code with spaces rather than tabs.
![image](https://user-images.githubusercontent.com/57483089/127336509-21636a6e-010d-4f1f-86e9-ca219132f385.png)
It all works just fine in the game like this, but since the standard is to use tabs instead it'd probably be good to adjust them:
```suggestion
/datum/reagent/consumable/ethanol/gimlet
	name = "Gimlet"
	id = "gimlet"
	description = "A sharp cocktail dating back to the 19th century. Gin and lime, nothing else."
	color = "#DEF8AB" // rgb (222, 248, 171)
	alcohol_perc = 0.3
	drink_icon = "gimlet"
	drink_name = "Gimlet"
	drink_desc = "There are debates on whether this drink should be half gin and half lime, or three parts gin and one part lime. All you know is, it's alcohol."
	taste_description = "sharpness"
```

Reply:
> You can't see it on the github diffs, but some of the new drinks are indented in the code with spaces rather than tabs.
> ![image](https://user-images.githubusercontent.com/57483089/127336509-21636a6e-010d-4f1f-86e9-ca219132f385.png)
> It all works just fine in-game like this, but since the standard is to use tabs instead it'd probably be good to adjust them:

fuck, im usually pretty good about using tabs. my bad. commiting suggestions, thank you, sabre.